feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/fee/utils/HydraDxQuoteSharedComputation.kt

```diff
@@ -35,31 +36,42 @@ class HydraDxQuoteSharedComputation(
         }
     }
 
-    suspend fun quote(
+    suspend fun paths(
         chain: Chain,
+        args: AssetExchangeQuoteArgs,
         accountId: AccountId,
-        fromAsset: Chain.Asset,
-        toAsset: Chain.Asset,
         scope: CoroutineScope
-    ): AssetExchangeQuote {
-        val key = "HydraDxQuote:${accountId.toHexString()}"
+    ): List<Path<HydraDxSwapEdge>> {
+        val key = "HydraDxPaths:${accountId.toHexString()}"
```

Comment:
The cache key should reflect all the arguments that might affect results
For example if you call the current code with different asset it will still returned old cached paths